src/main/java/org/cascadebot/cascadebot/commands/music/VolumeCommand.java

```diff
@@ -16,23 +16,21 @@ public void onCommand(Member sender, CommandContext context) {
             context.getTypedMessaging().replyInfo("Current volume is %s%%", player.getPlayer().getVolume());
             return;
         }
-        Integer Volume = null;
-        try {
-            Volume = context.getArgAsInteger(0);
-
-        } catch (java.lang.NumberFormatException e) {
+        int volume = -1;
+        if(context.isArgInteger(0)) {
+            volume = context.getArgAsInteger(0);
             context.getUIMessaging().replyUsage(this);
             return;
         }
-        if (Volume < 0 || Volume > 100) {
+        if (volume < 0 || volume > 100) {
```

Comment:
Why don't we just let them go 150 or smth?

Reply:
I think we can't. let me double check though.